### ISSUE TITLE
Fownloading HTML with a minus sign is read as a question-mark, different behaviour with Maven Surefire vs. running in IntelliJ

### ISSUE THREAD

**Issue by brampat** [HtmlUnit/htmlunit#252]
Example:
This page on [The Hacker News](https://thehackernews.com/2020/07/artificial-intelligence-training.html)
Has the following source:
`<meta content='Learn Machine Learning and AI – Online Training Program @ 93% OFF' itemprop='headline'/>`

It is read as:
`<meta content="Learn Machine Learning and AI ? Online Training Program @ 93% OFF" itemprop="headline"/>`

The interesting aspect is this. I've implemented a test using Spock & Groovy. The test:
* runs fine when run in IntelliJ
* fails when run using Maven

I will attach an example project in a few hours

**Comment by niloc132** [comment]
Looks like a charset problem - when running from maven can you confirm that `System.getProperty("file.encoding") is utf-8? Perhaps you can tell surefire to run with this set properly to resolve this.

**Comment by brampat** [comment]
Thanks, I tried to fiddle around with that but no result.
Of course I was also trying to isolate the problem into the example, which obviously didn't experience the issue. So now I'm trying to find the difference that causes the issue.

Thank

**Comment by brampat** [comment]
It may have something to do with the way I'm reading my test input-files

**Comment by brampat** [comment]
Okay so this is really annoying. I've been able to reproduce it in the isolated project when I run it under IntelliJ. I've uploaded the project to Github, then cloned the fresh repo and imported that as a separate project again in IntelliJ, ran the tests, no issues.
Ran all of my projects with the same test as mvn clean package from command-line and all of them work fine.

I'll just close the issue, since it's some kind of Maven / IntelliJ problem as far as I can see.
